graylog2-web-interface/src/pages/ShowDashboardPage.jsx

```diff
@@ -119,6 +119,9 @@ const ShowDashboardPage = React.createClass({
   _dashboardIsEmpty(dashboard) {
     return dashboard.widgets.length === 0;
   },
+  _validDimension(dimension) {
+    return dimension !== null && dimension !== undefined && dimension !== 0;
```

Comment:
The problem is that `Number.isInteger()` is not implemented in IE. I'll check if the polyfill we use supports it.